pkg/storage/note.go

```diff
@@ -57,6 +57,93 @@ func (s *Store) CreateNote(ctx context.Context, note *models.Note) (*models.Note
 	return s.FetchNoteByID(ctx, note.ID)
 }
 
+// UpdateNote updates all of a IT governance admin note's mutable fields.
+// The note's IsArchived field _can_ be set, though SetNoteArchived() should be used when archiving a note.
+func (s *Store) UpdateNote(ctx context.Context, note *models.Note) (*models.Note, error) {
+	stmt, err := s.db.PrepareNamed(`
+	UPDATE notes
+	SET
+		content = :content,
```

Comment:
Following on from the previous comment about a single mutation, you could add `is_archived = :is_archived` here.